dashboard.el

```diff
@@ -54,6 +55,7 @@ Return entire list if `END' is omitted."
   :syntax-table nil
   :abbrev-table nil
   (linum-mode -1)
+  (page-break-lines-mode 1)
```

Comment:
Yeah, it wasn't working for me before I added that line, but I'll try again to see if I had just gotten myself into a funny state.